bridges/ethElrond/bridgeExecutor.go

```diff
@@ -147,7 +147,11 @@ func (executor *bridgeExecutor) MyTurnAsLeader() bool {
 
 // GetBatchFromElrond fetches the pending batch from Elrond
 func (executor *bridgeExecutor) GetBatchFromElrond(ctx context.Context) (*clients.TransferBatch, error) {
-	return executor.elrondClient.GetPending(ctx)
+	batch, err := executor.elrondClient.GetPending(ctx)
```

Comment:
should we have 2 separate metrics for eth client as well?

Reply:
having 2 executors with MetricNumBatches each, this is already the expected behaviour